WebRoot/js/zimbraMail/core/ZmZimbraMail.js

```diff
@@ -2333,6 +2333,9 @@ function(parent, parentElement, adminUrl) {
 	    mi.addSelectionListener(new AjxListener(null, ZmZimbraMail.adminLinkCallback, adminUrl));
 	}
 
+	mi = menu.createMenuItem("standardHtmlLink", {text: ZmMsg.modernClient});
```

Comment:
can we change "standardHtmlLink" to modernLink

Reply:
Done